.rubocop.yml

```diff
@@ -43,7 +43,7 @@ Metrics/BlockLength:
 #   2. The models regarding the registry have to be modularized.
 #   3. LDAP has to be refactored.
 Metrics/ClassLength:
-  Max: 210
+  Max: 214
```

Comment:
Why ? 😕

Reply:
I add four lines to `spec_helper.rb`.
rubocop complained.

Reply:
Ah, I see. Then simply disable this cop for the whole cli.rb instead of raising this value. This is because this `portusctl` is deprecated in favor of [openSUSE/portusctl](https://github.com/openSUSE/portusctl) in 2.3, and since we have already created the 2.3 branch, we will nuke this code in master rather soon 😉